src/dams_mcda/server.R

```diff
@@ -248,14 +454,31 @@ server <- function(input, output, session) {
 		row.names(Dam1_Table) <- criteria_names
 		names(Dam1_Table) <- "Raw Score"
 
-
+    #results
+		# Figure 1 raw pref plot to show on output page
+		output$SummPlot1 <- renderBarPlot(

```

Comment:
this renderBarPlot is unnecessary because updateDamGraph function below it does the same thing